scipy/stats/_multivariate.py

```diff
@@ -2979,25 +2963,23 @@ def _rvs(self, n, shape, dim, df, C, random_state):
         """
         random_state = self._get_random_state(random_state)
         # Get random draws A such that A ~ W(df, I)
-        A = super()._standard_rvs(n, shape, dim, df, random_state)
+        A = self._inv_standard_rvs(n, shape, dim, df, random_state)
 
         # Calculate SA = (CA)'^{-1} (CA)^{-1} ~ iW(df, scale)
-        eye = np.eye(dim)
         trtrs = get_lapack_funcs(('trtrs'), (A,))
 
         for index in np.ndindex(A.shape[:-2]):
             # Calculate CA
-            CA = np.dot(C, A[index])
-            # Get (C A)^{-1} via triangular solver
+            # Get CA = (C A^{-1}).T via triangular solver
             if dim > 1:
-                CA, info = trtrs(CA, eye, lower=True)
+                CA, info = trtrs(A[index], C.T, lower=True, trans=True)
                 if info > 0:
                     raise LinAlgError("Singular matrix.")
                 if info < 0:
                     raise ValueError('Illegal value in %d-th argument of'
                                      ' internal trtrs' % -info)
             else:
-                CA = 1. / CA
+                CA = C / A[0]
             # Get SA
             A[index] = np.dot(CA.T, CA)
```

Comment:
Maybe a LAPACK trmm here would be faster. But `wishart_gen` performs a dense product as well.

Reply:
Benchmarking showed trmm was indeed faster, so I updated to use it.